### PR TITLE
Add test to check that image viewer is rendered on the exhibits show page

### PR DESCRIPTION
**Pull request by corylown**
Prevent a future regression of the issue reported in https://github.com/projectblacklight/spotlight/issues/2992 and fixed in https://github.com/projectblacklight/spotlight/pull/3060

### CODE REVIEW

File: spec/features/exhibits_show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+describe 'Exhibits show page', type: :feature do
```
Comment (jcoyne) [inline spec/features/exhibits_show_spec.rb:3]:
```suggestion
RSpec.describe 'Exhibits show page', type: :feature do
```
Avoid the need for monkeypatching.

**Review by jcoyne — CHANGES_REQUESTED** [review]
It seems to me this is the same scenario as is set up in https://github.com/projectblacklight/spotlight/blob/ec99378f29b42ba4ed742982cc77100054ec8190/spec/features/catalog_spec.rb#L50.  Can they be merged?

**Comment by corylown** [comment]
> . Can they be merged?

Yes, specs are combined now. Thank you.

File: spec/features/catalog_spec.rb [hunk 2]
```diff
@@ -25,6 +25,11 @@
       visit spotlight.search_exhibit_catalog_path(exhibit)
       expect(page).to have_selector '.card-header', text: 'Item visibility'
     end
+
+    it 'has an edit link' do
```
Comment (jcoyne) [inline spec/features/catalog_spec.rb:29]: Feature tests are relatively slow (especially in the setup).  Can we include the edit link assertion in the test above?  I think the feature is "it draws the page" and there are several assertions that support "the page is drawn correctly".

Comment (corylown) [reply spec/features/catalog_spec.rb:29]: The two admin tests are hitting different routes so I didn't think it made sense to combine them. The test above is for `search_exhibit_catalog_path` and the Edit link test I added is for `exhibit_solr_document_path`

Comment (jcoyne) [reply spec/features/catalog_spec.rb:29]: Oh, okay!

**Review by jcoyne — APPROVED** [review]

**Pull request merged**